doc/schemas/options_schema.json

```diff
@@ -48,12 +48,17 @@
       "additionalProperties": false,
       "properties": {
         "Resolution": {
-          "type": "integer",
+          "type": "array",
```

Comment:
Could we have an object with `width` and `height` fields here? I think it will be clear and it will ease debugging/checking with Wangcheck/interpreting these options.